docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -427,3 +427,54 @@ The Concourse CI doesn’t report health metrics through OpenTelemetry. However,
 https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor#span-metrics-processor[OpenTelemetry Collector Span Metrics Processor]
 to derive pipeline execution traces into KPI metrics like throughput and the error rate
 of pipelines.
+
+
+[discrete]
+[[check-service-health-from-cd-pipelines]]
+== Check service health from deployment pipelines
+
+
+Integrating automated service health checks in deployment pipelines is critical for end-to-end deployment automation, which crucially enables deployment frequency to be increased.
+
+Elastic Observability exposes service HTTP health check APIs that that integrate in deployment pipelines to verify the behaviour of newly deployed instances
+and automatically continue the deployments or rollback according to the health statuses.
+
+Example of a canary deployment pipeline leveraging Elastic health check HTTP APIs to automate the quality check before rolling out the deployment from the canary to the entire set of instances:
+
+image::images/ci-cd-canary-deployment-pipeline.png[Canary Deployment Pipeline]
+
+
+The primary HTTP API to check the health of a service is compare the transaction error rate with a threshold value.
+The API call requires authentication, we recomend to use an API Token to authenticate.
+The URL is `KIBANA_URL/internal/apm/services`, it is designed to query service metrics.
+In order to check the transaction error rate of the service on the newly deployed instance, we have to pass the following parameters to the API call:
+* `start` and `end`: time interval using the ISO-8601 format
```

Comment:
@cyrille-leclerc we can put the link to Wikipedia or the ISO (paid) if you want.

```suggestion
* `start` and `end`: time interval using the https://en.wikipedia.org/wiki/ISO_8601[ISO-8601] format
```